<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SOFIA Cruise Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="4" column="0">
     <widget class="QTabWidget" name="tabWidgetIO">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollAreaInput">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>765</width>
               <height>447</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="4" column="2" colspan="3">
               <widget class="QPlainTextEdit" name="plainTextEditNotes">
                <property name="plainText">
                 <string>This is a note.
It can be multiline.
It can have **markdown**
(also it can scroll)</string>
                </property>
               </widget>
              </item>
              <item row="5" column="4">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="2" colspan="3">
               <widget class="QLineEdit" name="lineEditGoal">
                <property name="text">
                 <string>Single line only! No special characters</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="labelAORNotes">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>AOR Notes:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelNotes">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Notes:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="labelAOR">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>AOR:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QLabel" name="labelAORFilename">
                <property name="font">
                 <font>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="text">
                 <string>No File Chosen ...</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="5">
               <widget class="QTableView" name="tableAORSummary"/>
              </item>
              <item row="5" column="2">
               <widget class="QPushButton" name="buttonAOROpen">
                <property name="text">
                 <string>Open ...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QLabel" name="labelLegSummary">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Leg Summary</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelGoal">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Goal:</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="5">
               <widget class="QPlainTextEdit" name="plainTextEditAORNotes">
                <property name="plainText">
                 <string>This is a note.
It can be multiline.
It can have **markdown**
(also it can scroll)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelInputSpacer1">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="5">
               <widget class="QTableView" name="tableLegSummary"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonPreviousLeg">
          <property name="text">
           <string>Previous Leg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonNextLeg">
          <property name="text">
           <string>Next Leg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOutput">
       <attribute name="title">
        <string>Output</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <layout class="QGridLayout" name="topGrid">
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEditUserName"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="labelUserName">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>User Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QTableWidget" name="tableFlightPlanSummary">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelInstruments">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Instrument</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxInstruments"/>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="labelFlightPlanFilename">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>No Flight Plan Chosen ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QLabel" name="labelFlightPlanImageFilename">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>No Flight Plan Image Chosen ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Flight_Plan"/>
    <addaction name="actionChoose_Flight_Plan_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen_Flight_Plan">
   <property name="text">
    <string>Open Flight Plan ...</string>
   </property>
  </action>
  <action name="actionChoose_Flight_Plan_Image">
   <property name="text">
    <string>Choose Flight Plan Image ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
